benchmark/benchmark.R

```diff
@@ -0,0 +1,179 @@
+library(MSstatsConvert)
+library(MSstats)
+library(ggplot2)
+library(dplyr)
+library(stringr)
+library(parallel)
+
+
+calculateResult <- function(summarized, label){
+  
+  model = groupComparison("pairwise", summarized)
+  comparisonResult <- model$ComparisonResult
+  
+  human_comparisonResult <- comparisonResult %>% filter(grepl("_HUMAN$", Protein))
+  
+  ecoli_comparisonResult <- comparisonResult %>% filter(grepl("_ECOLI$", Protein))
+  
+  yeast_comparisonResult <- comparisonResult %>% filter(grepl("_YEAST$", Protein))
+  
+  
+  human_median <- median(human_comparisonResult$log2FC, na.rm = TRUE)
+  ecoli_median <- median(ecoli_comparisonResult$log2FC, na.rm = TRUE) 
+  yeast_median <- median(yeast_comparisonResult$log2FC, na.rm = TRUE)
+  
+  cat("Expected Log Change Human:", human_median, "\n")
+  cat("Expected Log Change Ecoli:", ecoli_median, "\n")
+  cat("Expected Log Change Yeast:", yeast_median, "\n")
+  
+  #calculate SD and mean
+  
+  
+  # Kept the code for Individual Boxplots
+  
+  # boxplot(human_comparisonResult$log2FC,
+  #         main = "Boxplot of log2FC for Human",
+  #         ylab = "log2FC",
+  #         col = "lightblue")
```

Comment:
If you're not using the code, could you remove the comments?

Reply:
Fixed.